packages/roosterjs-editor-api/lib/format/insertTable.ts

```diff
@@ -3,6 +3,9 @@ import execFormatWithUndo from './execFormatWithUndo';
 
 const ZERO_WIDTH_SPACE = '&#8203;';
 
+/**
+ * The table format
+ */
```

Comment:
No parameter comments?

Reply:
I didn't add param comments for interfaces. @JiuqingSong Do we need to add to those?